### PR TITLE
DTSPO-4884 - Adding sds-release diagram

### PR DESCRIPTION
**Pull request by endakelly**

### CODE REVIEW

File: source/ways-of-working/common-pipeline/common-pipeline.html.md.erb [hunk 1]
```diff
@@ -47,6 +47,8 @@ For more information about Helm, please see the related new component
 
 ![The Common Pipeline](/images/common-pipeline.png)
 
+![SDS AKS Release](/images/sds-aks-release.png)
```
Comment (timja) [inline source/ways-of-working/common-pipeline/common-pipeline.html.md.erb:50]: this must be generic and can't have SDS in it's name

File: sds-aks-release.drawio [hunk 2]
```diff
@@ -0,0 +1 @@
...8x/4v0YN+q0HoBw0lfig96AcusAe1478tQQgGe0sQ5CdDXvqBC+x/yQd2m5PwqNnPjyXIvQ/sJQjxv0EQ+m2iO/myKdRPI8gDhfHL56qyf7xJn8L+eKdE1Z/UyfGzuVj/lFxW+qaJ9Reyp26HvyjOd60AebA58ztXgFhFUgAug4/2u9SAvEiQ71IDQpDMM5//7V0i32L1d4mDPmSqR/lbN0SFOcbV31Qx3zy75Eu6z8deoeiHe3hGEh8wBCdxloGijGEeRP9vh7z4Ob976+FHq/O7IYWvRAQ+QxIkco8sBIECn4er5LNc8WWk8DliPOi89ldjvn8j6C9T4yVhqcuzl41LvkiYpxnfPb80/gDDMUEUBI9oI1w/70GDjyHkDxT2qskpe1OegX6gMeTT5wWkve7SgH3sqPcm8474gH2HHJqHb/nL7FH7Xq0a/oG9uT8n3r5Y4/+yD8Mv0hD+l+kH/9X7LH6n0rbfmO9+4V0cqZcrvsAW+vOl7bfj6eeeRn81nr1J2/rW8d97z8fHuvdBq9p/DZW/NlQ+J7i+x972CH272c+vVKv+mIXuPW93NPwpdgmF3vgo7z1ijzoOU99Bv33W0fI+ZscNuI3QkIzorzdHWIrGvW8wOz5S/Rta+99N/0vi5Jv8ffK9pv8+xriLfHBCKsv0jhC/Yn7+3yXES3TxPpj1Xp23Hz/xr+rIQG8RwUvJxw+QGJ/dO+f3kBjfbiX/OInx+Inv86h+C4nx9YT48RIDsbZJNDXrhqEY1or/I1LE8mu2/nklML5aEHytbPlLBmWYt2CNeMCxAPh/wJlPvhz0wby9RLO//w4oX+HpfGtCf0GIPJYNN5Ik9CImfuhYowIm8uOHrPpZwn+Tc/Npxn/oNN+7MJ86kSAz1bib8DtB/Gp+Hyz8WyHy0gzktmvIo+Yg1yJ4+HKwz6yXFFENe4SoH8s9/2NEdZ8E0V9vMvk3yfJi275ZJvfezEdFxO/WKOQll+TfIuJfs4j4+7AcAHO3e87c890PLSBmHuDef+i2QN+JRCh+ty3Q19no74Y0vqK/wE9CGjc1H4/yrX4Q0ngIKr/Ct/ErAw3yLwj684DGw1m+d2HoQHb8tjjj66nyk3HGQ2I9cq3/CzN+bZjxzRz3U2HGY/P+gc33O8GMbyfRj4QZfzxXsb4KsXyqX8UX/wc=</diagram></mxfile>
```
Comment (timja) [inline sds-aks-release.drawio:1]: I'm not sure if `{{product}}` best describes this:
image

it's really `$product-$component` which might be a bit name, `$app`, or `$app-name` might better

This isn't handling the different cluster names / environments between SDS and CFT at all which will create confusion, e.g:

```
dev = preview
test = perftest
stg = aat
```

Step 3 is wrong:
image

it should be a PR image not a staging image

I think the diagram needs at least an intro as well not just dropping it on the page but open to suggestions

**Review by timja — CHANGES_REQUESTED** [review]

File: source/ways-of-working/common-pipeline/common-pipeline.html.md.erb (hunk 1 above)
Comment (endakelly) [reply source/ways-of-working/common-pipeline/common-pipeline.html.md.erb]: Updated

File: sds-aks-release.drawio (hunk 2 above)
Comment (timja) [reply sds-aks-release.drawio]: this includes pr number as well:
example `hmctspublic.azurecr.io/cmc/citizen-frontend:pr-3265-8cdc94e-20211005100511`

image

**Comment by timja** [comment]
> Should we mention anything about namespaces and any other information that comes from [https://github.com/hmcts/cnp-jenkins-config/blob/master/team-config.yml](https://github.com/hmcts/cnp-jenkins-config/blob/master/team-config.yml?rgh-link-date=2021-10-08T10%3A18%3A52Z) ?

good point, would be good to incorporate somehow. it is covered in team onboarding so _shouldn't_ be missing.

**Comment by endakelly** [comment]
> Should we mention anything about namespaces and any other information that comes from https://github.com/hmcts/cnp-jenkins-config/blob/master/team-config.yml ?

What sort of information were you thinking of?

File: .gitignore [hunk 3]
```diff
@@ -23,3 +23,6 @@
 
 Staticfile.auth
 vendor/
+
+# Ignore ruby
+.ruby-version
```
[the remark this answers is not in the document]
Comment (endakelly) [reply .gitignore]: It wasn't originally in the repo but created as a result of me using rbenv to run middleman, would it make it easier for others to get it up and running locally?

File: sds-aks-release.drawio (hunk 2 above)
Comment (endakelly) [reply sds-aks-release.drawio]: @njrich28 See section from here https://github.com/hmcts/hmcts.github.io/pull/129/files#diff-d429c650c892d8f9c7468cc618791c1cd630e7722bd857d82f0cd8554c53dbe1R51 

Does that make sense / reflect reality?

**Pull request merged**